PendingReleaseNotes.md

```diff
@@ -39,6 +39,9 @@
 - The flex driver can be configured to properly disable SELinux relabeling and FSGroup with the settings in operator.yaml.
 - Rgw is now configured with the Beast backend as of the Nautilus release
 - OSD: newly updated cluster from 0.9 to 1.0.3 and thus Ceph Nautilus will have their OSDs allowing new features for Nautilus
+- Rgw instances have their own key and thus are properly reflected in the Ceph status
+- The Rook Agent pods are now started when the CephCluster is created rather than immediately when the operator is started.
+- Ceph CRUSH tunable are not enforced to "firefly" anymore, Ceph picks the right tunable for its own version, to read more about tunable [see the Ceph documentation](http://docs.ceph.com/docs/master/rados/operations/crush-map/#tunables)
```

Comment:
This last item for pending release notes was the only one intended to add with this PR, right?

Reply:
Oops yes!